app/components/summary_section_component.rb

```diff
@@ -3,15 +3,28 @@
 class SummarySectionComponent < ViewComponent::Base
   renders_many :action_links
 
-  def initialize(title: , link: nil, link_title: nil, show_card:  true)
+  def initialize(title: , link: nil, link_title: nil, show_card:  true, show_icon: false, service_link: nil)
     super
     @title = title
     @link = link
     @link_title = link_title
     @show_card = show_card
+    @show_icon = show_icon
```

Comment:
here you can remove `show_icon` and jus use `service_link`, as `@show_icon = !service_link.nil?` , no ?

Reply:
I didn't understand what you meant exactly

Reply:
remove the argument `show_icon`, you don't need it, because `service_link` is enaugh